client/jetpack-cloud/components/sidebar/index.tsx

```diff
@@ -11,33 +21,55 @@ type Props = {
 	className?: string;
 	isJetpackManage?: boolean;
 };
-const Sidebar = ( { className, isJetpackManage = false }: Props ) => (
-	<nav className={ classNames( 'jetpack-cloud-sidebar', className ) }>
-		<Header forceAllSitesView={ isJetpackManage } />
-		<div className="jetpack-cloud-sidebar__main">
-			<ul role="menu" className="jetpack-cloud-sidebar__navigation-list">
-				<li
-					className={ classNames(
-						'jetpack-cloud-sidebar__navigation-item',
-						'jetpack-cloud-sidebar__navigation-item--highlighted'
-					) }
-				>
-					Navigation items
-				</li>
-				<li className="jetpack-cloud-sidebar__navigation-item">Will go here</li>
-			</ul>
-		</div>
-
-		<SiteSelector
-			showAddNewSite
-			showAllSites={ isJetpackManage }
-			isJetpackAgencyDashboard={ isJetpackManage }
-			className="jetpack-cloud-sidebar__site-selector"
-			allSitesPath="/dashboard"
-			siteBasePath="/landing"
-			wpcomSiteBasePath="https://wordpress.com/home"
-		/>
-	</nav>
-);
-
-export default Sidebar;
+const JetpackCloudSidebar = ( { className, isJetpackManage }: Props ) => {
```

Comment:
We might also want to discuss if we should put this component here (`jetpack-cloud/components/sidebar`) or in `components/jetpack/sidebar-v2` where the existing Jetpack sidebar resides.

Reply:
I think we can keep `jetpack-cloud/components/sidebar`

Reply:
I also lean more toward keeping it in `jetpack-cloud/components/sidebar` Thanks for sharing your thoughts. 👍

Reply:
I don't suppose it makes too much of a difference, but I created the `jetpack-cloud/components` directory because Jetpack Cloud is its own entry point/section in Calypso, which means components specific to it aren't loaded in other experiences. Because of this, in my mind, I made the low-cost decision to locate Jetpack Cloud-specific code inside the `jetpack-cloud` directory to keep experience-specific code together.